realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsUnmanagedTests.java

```diff
@@ -53,7 +53,7 @@ public void tearDown() {
         }
     }
 
-    // When unmanaged, an object's backlinks fields have their initialized value (probably null).
```

Comment:
@Zhuinden This is certainly legal, as things stand now: 

```
public class InitializedBacklinks extends RealmObject {
    private InitializedBacklinks child;

    @LinkingObjects("child")
    private final RealmResults<InitializedBacklinks> parents;

    public InitializedBacklinks() {
        this((RealmResults<InitializedBacklinks>) null);
    }

    public InitializedBacklinks(Realm realm) {
        this(realm.where(InitializedBacklinks.class).findAll());
    }

    private InitializedBacklinks(RealmResults<InitializedBacklinks> parents) {
        this.parents = parents;
    }

    public InitializedBacklinks getChild() {
        return child;
    }

    public void setChild(InitializedBacklinks child) {
        this.child = child;
    }
}

```

I think that might even be meaningful, when used Dynamically.